### PR TITLE
S24 fixing typos

### PR DESCRIPTION
**Pull request by dli505**
**Changes fixed in PR:**
- Fixed typos
- Fixed padding issue
Before:
image
After:
image

### CODE REVIEW

File: src/views/IDUploader/index.jsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ const IDUploader = () => {
             titleTypographyProps={{ align: 'center' }}
             className={styles.ID_card_header}
           />
-          <CardContent>
+          <CardContent sx={{ padding: 5 }}>
```
Comment (amos-cha) [inline src/views/IDUploader/index.jsx:95]: Probably best practice to put `css` adjustments within a .css file. However, this is a very isolated change so I think this is more of a preference for @EjPlatzer.

**Review by amos-cha — APPROVED** [review]
Besides the comment (probably just wait for Evan to respond), LGTM

File: src/views/IDUploader/index.jsx (hunk 1 above)
Comment (EjPlatzer) [reply src/views/IDUploader/index.jsx:95]: I think it's a matter of opinion which way is better in this case. I'm fine with merging this as is.

**Review by jsenning — COMMENTED** [review]
LGTM -- but, were you going to add graduation years for the Summer 2023 team?

**Comment by amos-cha** [comment]
> LGTM -- but, were you going to add graduation years for the Summer 2023 team?

I think that should just be in a different PR. No need to combine too many issues into a PR.

**Pull request merged**